### PR TITLE
Feature/add readme

### PR DESCRIPTION
**Pull request by donaldp**
## Updates
* Added a README file and a security policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Deta CLI
+
+The Deta CLI tool provides a number of helpful commands to help you with your Deta development.
+
+## Installation
+
+The installation guide for your Operating System can be found on the [Deta website](https://docs.deta.sh/docs/cli/install).
```
Comment (abdelhai) [inline README.md]:
```suggestion
The installation guide for your operating system can be found on the [Deta website](https://docs.deta.sh/docs/cli/install).
```
operating system is not capitalized

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Documentation for the Deta CLI can be found on the [Deta website](https://docs.deta.sh/docs/cli/commands).
+
+## Security Vulnerabilities
+
+Please review [our security policy](https://github.com/deta/deta-cli/security/policy) on how to report security vulnerabilities.
+
+## License
+
+The MIT License (MIT). Please see [License File](LICENSE) for more information.
```
Comment (abdelhai) [inline README.md]:
```suggestion
The MIT License (MIT). Please see the [license file](LICENSE) for more information.
```

**Review by abdelhai — CHANGES_REQUESTED** [review]
looks good, just minor things

**Comment by donaldp** [comment]
@abdelhai updated 👍

**Review by abdelhai — APPROVED** [review]

**Pull request merged**